src/test/java/com/redhat/cloud/common/clowder/configsource/ConfigSourceTest.java

```diff
@@ -485,24 +490,24 @@ void testKafkaSslUsingCommonCa() {
         assertFalse(ccs2.getPropertyNames().contains(CAMEL_KAFKA_SSL_TRUSTSTORE_LOCATION_KEY));
         assertFalse(ccs2.getPropertyNames().contains(CAMEL_KAFKA_SSL_TRUSTSTORE_TYPE_KEY));
 
-        final ClowderConfigSource ccs3 = new ClowderConfigSource("target/test-classes/cdappconfig_kafka_ssl_sec_type_common_ca.json", APP_PROPS_MAP, true);
+        final ClowderConfigSource ccs3 = configSourceWithFile("/cdappconfig_kafka_ssl_sec_type_common_ca.json", true);
 
         assertFalse(ccs3.getPropertyNames().contains(KAFKA_SASL_JAAS_CONFIG_KEY));
         assertFalse(ccs3.getPropertyNames().contains(KAFKA_SASL_MECHANISM_KEY));
         assertTrue(ccs3.getPropertyNames().contains(KAFKA_SECURITY_PROTOCOL_KEY));
-        assertFalse(ccs3.getPropertyNames().contains(KAFKA_SSL_TRUSTSTORE_LOCATION_KEY));
-        assertFalse(ccs3.getPropertyNames().contains(KAFKA_SSL_TRUSTSTORE_TYPE_KEY));
+        assertTrue(ccs3.getPropertyNames().contains(KAFKA_SSL_TRUSTSTORE_LOCATION_KEY));
+        assertTrue(ccs3.getPropertyNames().contains(KAFKA_SSL_TRUSTSTORE_TYPE_KEY));
```

Comment:
The JSON file does not contain cert data. We're not creating a temp cert file so I don't think we should expose these properties.

Same a few lines below.

Reply:
I've reverted all the assert changes I made, so we ensure we don't change the existing behaviour.